tests/integration/tiles/tiles.spec.ts

```diff
@@ -46,80 +46,140 @@ describe('tiles', function () {
     });
 
     describe('Happy Path', function () {
-      it('should return ok', async function () {
-        const bbox = getBbox();
+      describe('POST /tiles/bbox', function () {
+        it('should return ok', async function () {
+          const bbox = getBbox();
 
-        const response = await requestSender.postTilesByBboxRequest(bbox, 0, 1);
+          const response = await requestSender.postTilesByBboxRequest(bbox, 0, 1);
 
-        expect(response.status).toBe(httpStatusCodes.OK);
-        expect(response).toSatisfyApiSpec();
+          expect(response.status).toBe(httpStatusCodes.OK);
+          expect(response).toSatisfyApiSpec();
+        });
+      });
+
+      describe('POST /tiles/list', function () {
+        it('should return ok', async function () {
+          const response = await requestSender.postTilesList([{ z: 0, x: 0, y: 0, metatile: 8 }]);
+
+          expect(response.status).toBe(httpStatusCodes.OK);
+
+          expect(response).toSatisfyApiSpec();
+        });
       });
     });
     describe('Bad Path', function () {
```

Comment:
test every different case where an error is invoked by `validateTile`

Reply:
testing every case is wrong, they are already covered by validateTile tests.